plugins/kubernetes/app/views/kubernetes_project/show.html.erb

```diff
@@ -1,10 +1,30 @@
 <%= render 'projects/header', project: @project, tab: 'kubernetes' %>
 
-
-<section class="tabs">
-  <h1>Project Roles</h1>
-  <%= render partial: 'kubernetes_roles/roles_list_show' %>
-
-  <h1>Kubernetes Releases</h1>
-  <%= render partial: 'kubernetes_release_groups/kuber_release_group_table' %>
```

Comment:
They're still being used by the pages that get rendered when you access directly to: `/kubernetes_release_groups` and  `/kubernetes_roles`, e.g., following a `create`.

But yes, they're going to be removed soon.